lib/scraper/resistance_calendar.rb

```diff
@@ -0,0 +1,36 @@
+require 'scraper/scraper_base'
+require 'hyperclient'
+require 'cta_aggregator_client'
+
+module Scraper
+  class ResistanceCalendar < ScraperBase
+
+    ORIGIN_SYSTEM = "Resistance Calendar"
+    SYSTEM_NAME = "resistance-calendar"
+    ORIGIN_URL = "https://resistance-calendar.herokuapp.com/v1/events"
+    EVENT_ATTRS = [
+      'browser_url', 'origin_system', 'title', 'description', 'start_date', 'end_date', 'free', 'featured_image_url', 'identifiers'
+    ].freeze
+    
+    def scrape
+      osdi = Hyperclient.new(ORIGIN_URL)
```

Comment:
I haven't played with Hyperclient before.  What's the big benefit over similar libraries?

Reply:
I hadn't previously either--I used it because it's referenced in the OSDI docs (search 'Hypermedia'  [here](https://opensupporter.github.io/osdi-docs/)). It seems OSDI implements a 'hypermedia' standard, which as far as I can tell translates to "over-architected and unnecessarily complex API." It does seem to consume OSDI without much fuss though :-)

Reply:
Ahh, I totally missed that in the OSDI Docs.